nixos/modules/services/security/aide.nix

```diff
@@ -0,0 +1,145 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.aide;
+  configFile = pkgs.writeText "aide.conf" ''
+    # The location of the database
+    database=${cfg.database_url}
+
+    # The location of the log file
+    log_file=/var/log/aide/aide.log
+
+    # Directories to be checked
+    ${lib.concatStringsSep "\n" (map (dir: ''
+    ${dir} NORMAL
```

Comment:
When trying to evaluate the configuration, I get the error: group 'NORMAL' is not defined, which is true according to the [GROUPS section in the man page](https://github.com/aide/aide/blob/v0.18.x/doc/aide.conf.5#L403-L421)